src/index.html

```diff
@@ -3,11 +3,38 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section class="card" data-qa="card">
+      <img
+      class="card__image"
+      src="./images/imac.jpeg"
+      alt="Apple iMac">
```

Comment:
```suggestion
      <img
        class="card__image"
        src="./images/imac.jpeg"
        alt="Apple iMac"
      >
```
fix your code styles according to style guide

fix all such cases